dipy/reconst/shore.py

```diff
@@ -750,7 +761,8 @@ def shore_indices(radial_order, index):
     m_i = 0
 
     if n_c < (index + 1):
-        msg = "The index is higher than the number of coefficients of the truncated basis."
+        msg = "The index is higher than the number of coefficients of the "
+        msg += "truncated basis."
```

Comment:
Here again it would be helpful to say what it received as input, and how it differs from the expected value (I know you only changed the line-wrapping here, but if we can improve it, why not?).